src/mod2c_core/nocpout.c

```diff
@@ -1950,7 +1950,7 @@ void bablk(ba, type, q1, q2)
 "  _thread = _ml->_thread;\n"
 "  double * _nt_data = _nt->_data;\n"
 "  double * _vec_v = _nt->_actual_v;\n"
-"  int stream_id = _nt->stream_id;\n"
+"  int stream_id = _nt->streams[_nt->stream_id];\n"
 "  #if LAYOUT == 1 /*AoS*/\n"
```

Comment:
@iomaganaris : remind me better - MOD2C remains only OpenACC (and not OpenMP). So does this change required in MOD2C i.e. using `_nt->stream_id` as async launch stream for OpenACC works fine right? 🤔

Reply:
It does, yes. I just changed this to be similar to the form needed by `OpenMP` for homogenity

Reply:
Ok.